### DISCUSSION TITLE
Please add source from app Kiedy Śmieci

### DISCUSSION THREAD

**Discussion by Sivv3k** [mampfes/hacs_waste_collection_schedule#3141] [Ideas]
Please add collection from app Kiedy Śmieci. IOS app https://apps.apple.com/pl/app/kiedy-%C5%9Bmieci/id1539957094?l=pl or Android https://play.google.com/store/apps/details?id=com.fxsystems.KiedySmieci_info&pli=1. The web version is available at https://cloud.fxsystems.com.pl/OdbiorySmieci/Dostepnosc.dll

BR

**Comment by 5ila5** [comment]
please open an issue here: https://github.com/mampfes/hacs_waste_collection_schedule/issues/new?assignees=&labels=source+request&projects=&template=SOURCE-REQUEST.yml&title=%5BSource+Request%5D%3A+
